apps/angular/src/app/features/anime/anime-management/anime-form/anime-form.component.html

```diff
@@ -0,0 +1,104 @@
+<form [formGroup]="formGroup" (submit)="handleSubmit()" class="anime-form">
+
+    <div class="anime-form-section">
+        <mat-form-field class="anime-form-section__field" *ngIf="formGroup.controls.englishTitle as englishTitle">
```

Comment:
```suggestion
        <mat-form-field 
          *ngIf="formGroup.controls.englishTitle as englishTitle"
          class="anime-form-section__field"  
        >
```

Try to write structural directives at the beginning of tag attributes.